roles/edpm_ovn_bgp_agent/tasks/configure.yml

```diff
@@ -14,14 +14,45 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Ensure the Openvswitch package is installed
+  ansible.builtin.package:
+    name: openvswitch
+    state: present
+
+- name: Ensure the OVS service is running
+  ansible.builtin.systemd:
+    name: openvswitch
+    state: started
+
```

Comment:
I've seen these tasks in, at least, 3 roles. I mean, they're correct, but maybe we could choose a role, move these tasks into a single tasks file and use `include_role/tasks_from` directives to avoid code duplication.